packages/manager/apps/pci-instances/src/pages/instances/action/RescueAction.page.tsx

```diff
@@ -15,15 +15,17 @@ import { imagesRescueSelector } from '@/data/hooks/image/selector/image.selector
 
 type TRescueActionPageProps = 'rescue/start' | 'rescue/end';
 
-export const RescueActionPage: FC<{
+export type TCustomModalActionPage = {
```

Comment:
I would recommend to rename `TCustomModalActionPage` by  `TRescueActionPageProps` as there are props for the component.
BTW, handler callbacks functions should begin with `on...`

```ts
onMutationError: (error: unknown) => void;
...
```